packages/api/package.json

```diff
@@ -25,6 +25,7 @@
   "license": "ISC",
   "dependencies": {
     "compression": "^1.7.1",
+    "core-js": "^2.5.0",
```

Comment:
This was necessary to solve some version mismatch problem in `babel-polyfill` after updating dependencies.